docs/querying/sql-query-context.md

```diff
@@ -60,19 +60,31 @@ For more information, see [Overriding default query context values](../configura
 
 ## Set the query context
 
-You can configure query context parameters in the `context` object of the [JSON API](../api-reference/sql-api.md) or as a [JDBC connection properties object](../api-reference/sql-jdbc.md).
+You can configure query context parameters in the `context` object or as part of the `query` string with [`SET`
```

Comment:
This has become kind of a run-on. Perhaps splitting it up into multiple sentences or a bulleted list would be good.

Reply:
reworked for clarity